src/item.h

```diff
@@ -2121,6 +2124,7 @@ class item : public location_visitable<item>, public game_object<item>
         time_duration age() const;
         void set_age( const time_duration &age );
         void legacy_fast_forward_time();
+        bool _active() const;
```

Comment:
```suggestion
        bool is_active() const;
```

is_active seems to be more consistent.